mininterface/_tk_interface/utils.py

```diff
@@ -67,12 +69,55 @@ def get(self):
 
 
 def choose_file_handler(variable: Variable, tag: PathTag):
+    """Handler for file/directory selection on PathTag"""
     def _(*_):
-        if tag.multiple:
-            out = str(list(askopenfilenames(title="Choose files")))
+        initialdir = tag.val if tag.val else os.getcwd()
+
+        # Check whether this is a directory selection
+        if tag.is_dir:
+            # Directory selection using askdirectory
+            kwargs = {"title": "Select Directory", "initialdir": initialdir}
+            selected_dir = askdirectory(**kwargs)
```

Comment:
It is not now possible to select multiple directories. But leave it like that and leave a note `# NOTE this should support multiple dirs` pls